data/json/monstergroups/wilderness.json

```diff
@@ -1825,7 +1826,44 @@
       { "monster": "mon_antlion_giant", "weight": 2, "cost_multiplier": 10 },
       { "monster": "mon_stag_beetle_small", "freq": 2, "cost_multiplier": 0, "starts": 168, "ends": 720 },
       { "monster": "mon_stag_beetle_small", "freq": 2, "cost_multiplier": 0, "starts": 168 },
-      { "monster": "mon_stag_beetle_giant", "freq": 1, "cost_multiplier": 0, "starts": 720 }
+      { "monster": "mon_stag_beetle_giant", "freq": 1, "cost_multiplier": 0, "starts": 720 },
+      { "monster": "mon_caterpillar", "weight": 3, "cost_multiplier": 0 },
+      { "monster": "mon_woodlouse", "weight": 3, "cost_multiplier": 0 },
+      { "monster": "mon_caterpillar_giant", "weight": 2, "cost_multiplier": 0 },
```

Comment:
Needs a start timer (preferably shorter than the _giant mobs).

Reply:
added a 120 hours time for the giant caterpillars.